### PR TITLE
Automatic tests by Travis CI

### LINKED ISSUE #11: Setup Travis
It seems to me to be the most popular free testing service with much possible customization (a fully fledged VM). Drawback might be only Ubuntu 12.04 or 14.04 LTS Server Edition 64 bit are available, i.e. complicating setup progress or not testing with actual deployment environment.

### PR DESCRIPTION
**Pull request by timkol**
Closes #11

### CODE REVIEW

**Comment by Werkov** [comment]
I have no idea why database tests fail when they run in parallel (there are up to 8 databases and access to each one of them is serialized).
Similarly, I serialized the tests that share common upload directory.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+install:
+#  - composer install --no-interaction --prefer-source
+  - "sed -e 's/user:/user: root/' app/config/config.local.neon.sample > app/config/config.local.neon"
+  - ./i18n/compile.sh
+
+before_script:
+  - mysql -e 'CREATE DATABASE newfks;'
+  - mysql newfks < sql/schema.sql
+  - mysql newfks < sql/views.sql
+  - mysql newfks < sql/initval.sql
```
Comment (Werkov) [inline .travis.yml]: Is the `newfks` database used anywhere? AFAIK all databases for tests are created via `init-database.sh` script.

File: tests/Events/Model/ApplicationHandler.fyziklani.phpt [hunk 2]
```diff
@@ -88,6 +88,7 @@ class ApplicationHandlerTest extends EventTestCase {
             array(
                 'name' => $teamName,
                 'phone' => '',
+		'force_a' => false,
```
Comment (Werkov) [inline tests/Events/Model/ApplicationHandler.fyziklani.phpt]: Is this wrongy indented? (Sorry for nitpicking.)

**Comment by timkol** [comment]
@Werkov Database tests fail (that SQL error on constraint) because when (another) test fails on exception, tearDown() method of that test is not executed, so when this test uses database, it isn't cleaned up after use. This bug was addressed and solved in nette/tester#254

File: tests/Events/Model/ApplicationHandler.fyziklani.phpt (hunk 2 above)
Comment (timkol) [inline tests/Events/Model/ApplicationHandler.fyziklani.phpt]: Now it should be fixed. Thanks

File: .travis.yml (hunk 1 above)
Comment (timkol) [inline .travis.yml]: Fixed.

**Comment by Werkov** [comment]
Mergnul jsem to, ať máme další merge do mastera už automaticky testované (doufám, že to tak bude fungovat).
Pro paralelizaci testů založím další issue.

**Pull request merged**